### PR TITLE
Fix JSON formatting output

### LINKED ISSUE #9: Fix JSON format
When requesting secrets with JSON format type invalid JSON string returns:
```
"{\"USERNAME\":\"jsonUser\",\"PASSWORD\":\"jsonPassword\"}"
```

### PR DESCRIPTION
**Pull request by yoyrandao**

### CODE REVIEW

**Comment by yoyrandao** [comment]
Closes #9

File: src/SecretsService/Services/Formatters/JsonSecretFormatter.cs [hunk 1]
```diff
@@ -9,16 +9,16 @@ namespace Keebox.SecretsService.Services.Formatters
 {
 	public class JsonSecretFormatter : ISecretFormatter
 	{
-		private readonly ISerializer? _serializer;
-
-		public JsonSecretFormatter(ISerializer? serializer) =>
+		public JsonSecretFormatter(ISerializer? serializer)
```
Comment (TheAsda) [inline src/SecretsService/Services/Formatters/JsonSecretFormatter.cs]: Nullable?

File: src/SecretsService/Services/Formatters/XmlSecretFormatter.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public class XmlSecretFormatter : ISecretFormatter
 		public XmlSecretFormatter(ISerializer? serializer) =>
```
Comment (TheAsda) [inline src/SecretsService/Services/Formatters/XmlSecretFormatter.cs]: Body

**Review by TheAsda — CHANGES_REQUESTED** [review]

File: src/SecretsService/Services/Formatters/JsonSecretFormatter.cs (hunk 1 above)
Comment (yoyrandao) [reply src/SecretsService/Services/Formatters/JsonSecretFormatter.cs]: done

File: src/SecretsService/Services/Formatters/XmlSecretFormatter.cs (hunk 2 above)
Comment (yoyrandao) [reply src/SecretsService/Services/Formatters/XmlSecretFormatter.cs]: done

**Review by TheAsda — APPROVED** [review]

**Pull request merged**